composer.json

```
@@ -35,7 +35,7 @@
 	"scripts": {
 		"compat": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --standard=phpcompat.xml.dist --report=summary,source",
 		"format": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf --report=summary,source",
-		"lint": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source",
+		"lint": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source --basepath='./'",
```

```suggestion
		"lint": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source --basepath=./",
```

The quotes shouldn't be needed.